### DISCUSSION TITLE
Get both transactions & transfers of an user using a single query

### DISCUSSION THREAD

**Discussion by sadaimudiNaadhar** [bavix/laravel-wallet#310] [Q&A] [answered]
Hi how can i get both transactions & transfers using a single query? Is there any easy way of achieving this? I need this feature to display the transfers & deposits separately in a single section. Could you please help me?

**Comment by rez1dent3** [comment, marked as answer]
Hello. #240 #203 #198

**Reply by sadaimudiNaadhar** [reply]
Thanks, But actually I need to get both transfers & transactions, I had seen two separate methods for this in your trait. But am asking like how can I know a transaction is **transfer**  ? Like a combined use of **$wallet->transactions() JOIN $wallet->transfers()**

My requirement is to combine both functions

![image](https://user-images.githubusercontent.com/42279368/111515260-9163e380-8778-11eb-8324-f921beccf645.png)

Is there an easy way to achieve this?

**Reply by sadaimudiNaadhar** [reply]
Would be useful, so that i can avoid hard coding table columns outside your package scope

**Reply by rez1dent3** [reply]
Unfortunately, you won't succeed. Transfers and transactions are different tables.
When writing to transfer, two records are written to the transactions table.

I would advise you to add meta-information when transferring information, then it will be easier for you to display on the frontend.
